src/wazuh_modules/wm_ciscat.c

```diff
@@ -448,12 +468,11 @@ void wm_ciscat_run(wm_ciscat_eval *eval, char *path, int id, const char * java_p
     }
 
     // Create arguments
-
-    wm_strcat(&command, ciscat_script, '\0');
-
-    // Accepting Terms of Use
-
-    wm_strcat(&command, "-a", ' ');
+    strcpy(tmp, path);
+    strcat(tmp, "/");
+    
+    wm_strcat(&command, tmp, '/');    
+    wm_strcat(&command, ciscat_binary, '\0');
```

Comment:
This is an insecure operation as we're copying `path`, with an undefined size, into an `OS_MAXSTR` bytes string.
```suggestion
    wm_strcat(&command, path, '/');
    wm_strcat(&command, ciscat_binary, '/');
```